Frends.FTP.DownloadFiles/Frends.FTP.DownloadFiles/Logging/FtpLogger.cs

```diff
@@ -5,182 +5,179 @@
 using Frends.FTP.DownloadFiles.Enums;
```

Comment:
https://github.com/FrendsPlatform/Frends.FTP/blob/3b2b2fbd96e793fb58e0ddc3e006dc3f6fcb985c/Frends.FTP.DownloadFiles/Frends.FTP.DownloadFiles/Logging/FtpLogger.cs#L1
You are not using this namespace, so it can be removed.

https://github.com/FrendsPlatform/Frends.FTP/blob/3b2b2fbd96e793fb58e0ddc3e006dc3f6fcb985c/Frends.FTP.DownloadFiles/Frends.FTP.DownloadFiles/Logging/FtpLogger.cs#L3
You are not using this namespace, so it can be removed.